testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsf/testurl/web.xml

```diff
@@ -37,7 +37,7 @@
    <!-- 
       FacesServlet is main servlet responsible to handle all request. 
       It acts as central controller.
-      This servlet initializes the JSF components before the JSP is displayed.
+      This servlet initializes the JSF components before the Jakarta Server Page is displayed.
```

Comment:
I checked the [Jakarta Server Pages 3.0 spec](https://jakarta.ee/specifications/pages/3.0/jakarta-server-pages-spec-3.0.html) and they do not define the term `Jakarta Server Page`.  They do seem to loosely use the terms { `page`, `pages`, `Page`,  `Pages` } and still also refer to `JSP page`.  IMO, we should do our own thing, as the spec needs work.

How about changing to:
`This servlet initializes the Jakarta Server Faces components before the Jakarta Server Pages page is displayed.`
OR
`This servlet initializes the JSF components before the Jakarta Server Pages page is displayed.`

Reply:
Okay! I'll do the changes.

Reply:
Thanks! @scottmarlow Did the changes :)

Reply:
Thanks @RanabirChakraborty for the +10000 super effort!  Very much appreciated!

Reply:
@scottmarlow I'm happy to be a part of it :) and anything for the community :)